src/pudl/scripts/dbt_helper.py

```diff
@@ -108,23 +108,10 @@ def add_column_tests(self, column_tests: dict[str, list]) -> "DbtSource":
 
         return self.model_copy(update={"columns": list(columns.values())})
 
```

Comment:
Both `add_source_tests` and `add_column_tests` return `DbtTable` objects (since they're making an updated copy the model they're part of) but they claim to be returning `DbtSource` objects, and those types aren't compatible with each other.